src/clr-angular/utils/host-wrapping/empty-anchor.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {Component} from "@angular/core";
+
+@Component({
+    selector: "clr-empty-anchor",
```

Comment:
Technically this is unnecessary, so would be good to remove it unless we actually want to use it.